src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -69,7 +73,9 @@ export class FeederPodcastModel extends BaseModel {
     enclosurePrefix: [URL('Not a valid URL')],
     summary: [LENGTH(0, 4000)],
     displayEpisodesCount: [POSITIVE_INT_OR_BLANK],
-    displayFullEpisodesCount: [POSITIVE_INT_OR_BLANK]
+    displayFullEpisodesCount: [POSITIVE_INT_OR_BLANK],
+    paymentPointer: [URL('Not a valid URL')],
+    donationUrl: [URL('Not a valid URL')]
```

Comment:
Are both of these fields _always_ expected to be URLs?

Reply:
Yes. I think we can expect that. The payment pointer should be a link to the wallet.